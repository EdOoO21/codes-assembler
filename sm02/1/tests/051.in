717 28 131 -669 -497 681 -527 705 131 -477 897 776 -187 122 982 -51 498 425 122 72 288 -299 200 352 -263 -208 -432 -173 427 -985 746
