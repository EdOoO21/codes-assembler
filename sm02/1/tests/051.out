746
-985
427
-173
-432
-208
-263
352
200
-299
288
72
122
425
498
-51
982
122
-187
776
897
-477
131
705
-527
681
-497
-669
131
28
717
